program/tests/helpers/common.rs

```diff
@@ -43,69 +67,6 @@ pub fn init_mollusk() -> Mollusk {
     mollusk
 }
 
-pub const MINT_DECIMALS: u8 = 12;
-pub const MINT_SUPPLY: u64 = 500_000_000;
-pub const FREEZE_AUTHORITY: Pubkey =
-    Pubkey::from_str_const("11111115q4EpJaTXAZWpCg3J2zppWGSZ46KXozzo9");
-
-fn token_2022_with_extension_data(supply: u64) -> Vec<u8> {
```

Comment:
Here and down represents all of the individual mint and token account creators that are now replaced